.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
Please add it to the gitignore. This file from you VS Code only, not from project